src/brs/GeneratorImpl.java

```diff
@@ -134,15 +136,25 @@ public BigInteger calculateHit(long accountId, long nonce, byte[] genSig, byte[]
   }
 
   @Override
-  public BigInteger calculateDeadline(long accountId, long nonce, byte[] genSig, int scoop, long baseTarget, int blockHeight) {
-    return burstCrypto.calculateDeadline(accountId, nonce, genSig, scoop, baseTarget, getPocVersion(blockHeight));
+  public BigInteger calculateDeadline(BigInteger hit, long baseTarget, int blockHeight) {
```

Comment:
New deadline as described on the analysis article https://link.medium.com/LcwXJx8Yn3